Cauli/Response.swift

```diff
@@ -39,4 +39,31 @@ public struct Response: Codable {
     }
 
     private var urlResponseRepresentable: URLResponseRepresentable
+
+    public var suggestedFilename: String {
+        // TODO: Check the Content-Disposition header for a possible filename
```

Comment:
there's a todo :).

Sorry, not really PR related, but let's start the discussion and if we like it I'll create a ticket.
What do you think about a TODO violation rule within our CI process? My proposal: whenever someone writes a TODO comment without an issue number behind it, the build should fail. So that line would cause a build error, but `// TODO #152` would build

Reply:
Since that PR is WIP, that rule should be ignored.

Reply:
I like the idea of adding a swiftlint rule for the todo violations. But I think we shouldn't add a specific rule once we reference an issue. I would suggest once you create an issue for a todo, it is better to just have the issue. This way we have one defined way where *all* the issues and todos are and we ensure it via swiftlint. What do you think about that?

Reply:
I'll write a ticket for that.